### DISCUSSION TITLE
查询偶发报错：field name is invalid. field=

### DISCUSSION THREAD

**Discussion by zenoyang** [apache/doris#7122] [A - General / Q&A]
Doris版本： 0.13

报错内容：field name is invalid. field= 字段名称

我们内部多个集群，不同的表都出现了这个问题。 同样的查询，偶尔会报这样的错误。 无法稳定复现，sql、字段、表模型也没有什么规律 : (

**Comment by morningman** [comment]
Better upgrade to 0.14 before go further debug.
